src/nsrt_learning.py

```diff
@@ -39,10 +39,16 @@ def learn_nsrts_from_data(dataset: Dataset, predicates: Set[Predicate],
 
     # Now that options are learned, we can update the segments to include
     # which option is being executed within each segment.
-    for partition, option_spec in zip(partitions, option_specs):
-        for (segment, _) in partition:
-            # Modifies segment in place.
-            _update_segment_from_option_specs(segment, option_spec)
+    # If not option learning, the segments will already have the options
+    # after segmentation, so this step can be skipped.
+    assert not CFG.do_option_learning
```

Comment:
Maybe in the next PR, we should add a dummy option learning that just pulls in the ground truth options? And then we can do this. I'm assuming you kept it this way for testing reasons.

Reply:
In fact, I don't think we're testing the option learning code path at all right now.